src/components/StopwatchEntry.vue

```diff
@@ -55,15 +55,21 @@ export default {
       let new_event = JSON.parse(JSON.stringify(this.event));
       new_event.data.running = false;
       new_event.duration = (moment() - moment(new_event.timestamp)) / 1000;
+      new_event.data.tags = this.editTags(new_event.data.tags);
```

Comment:
Nitpick, but would prefer if this line was above the previous one, so that assigns to `data` are gouped.